command: command to allow changing mclock profile (#2)
red-hat-storage/odf-cli: docs/set.md

# Set

The set command supports the following sub-commands:

> * [Recovery-profile](#recovery-profile)

Reviewer: ```suggestion
* [recovery-profile](#recovery-profile)
```

Author: this auto correct by vs code, probably due to `markdown` extension

Author: never-mind, I found the fix

> ## Recovery-profile

Reviewer: ```suggestion
## recovery-profile
```

This command will set the recovery profile to favor either the new client IO, recovery IO, or a balanced mode. The default is `balanced` mode.
We can use the following built-in profile types:

> * balanced(default)

Reviewer: Let's not mention default here since it's already mentioned in the paragraph.
```suggestion
* balanced
```

* high_client_ops
* high_recovery_ops

> ### Example

Reviewer: ```suggestion
```

```code
odf set recovery-profile high_client_ops
```

> To verify the recovery profile setting run [this command](get.md#recovery-profile).

Reviewer: ```suggestion
To verify the recovery profile setting run [odf get recovery-profile](get.md#recovery-profile).
```

> To verify the mclock recovery profile setting run [this command](get.md#recovery-profile).

Reviewer: Let's not mention mclock, it's a detail the ODF users won't care about.
```suggestion
To verify the recovery profile setting run [this command](get.md#recovery-profile).
```

# Set

The set command supports the following sub-commands:

* [Recovery-profile](#recovery-profile)

## Recovery-profile

This command is used choose/modify the [mclock](https://docs.ceph.com/en/quincy/rados/configuration/mclock-config-ref/#mclock-profiles-definition-and-purpose) scheduler profile that is being used during recovery.

We can use the following built-in profile types:

- [balanced](https://docs.ceph.com/en/quincy/rados/configuration/mclock-config-ref/#balanced-default) (default)
- [high_client_ops](https://docs.ceph.com/en/quincy/rados/configuration/mclock-config-ref/#high-client-ops)

> - [high_recovery_ops](https://docs.ceph.com/en/quincy/rados/configuration/mclock-config-ref/#high-recovery-ops)

Reviewer 2: Should these link to RHCS docs?

Author: I was not able to point url to each individual topics
